app/Http/Controllers/CityController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\City;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+
+class CityController extends Controller
+{
+    public function index()
+    {
+        return view('cities.index');
+    }
+
+    public function getAll()
+    {
+        $cities = City::all();
+        return response()->json(['cities' => $cities]);
+    }
+
+    public function store(Request $request): JsonResponse
+    {
+        $validator = Validator::make(
+            $request->all(),
+            [
+                'name' => 'required|string|max:255|unique:cities,name',
```

Comment:
```suggestion
                'name' => ['required', 'string', 'max:255', 'unique:cities,name'],
```

This will be in FormRequest rules but anyways we prefer to write rules as an array instead of using | as separation